tests/basic_usage.rs

```diff
@@ -32,6 +32,19 @@ actions:
     to: mydircopy
 "#,
                 ),
+                f(
+                    "where_condition.yaml",
+                    r#"
+where:
+  - non.existing.variable == true
```

Comment:
This seems like the old revision with multiple

Reply:
Argh. Thanks. That's what reviews are for!